patroni/api.py

```diff
@@ -75,6 +75,7 @@ def _write_status_response(self, status_code, response):
             lost = patroni.logger.records_lost
             if lost:
                 response['logger_records_lost'] = lost
+        response['action'] = self.server.patroni.ha.action
```

Comment:
I don't think we really need to introduce one more field in the response.
During reinitialize the state field is set to `creating replica`.